examples/MicroserviceExample/docker-compose.yml

```diff
@@ -4,6 +4,16 @@ services:
     ports:
       - 9411:9411
 
+  otel-collector:
```

Comment:
do we need collector to forward to Zipkin, given Zipkin can natively accept OTLP?

Reply:
Zipkin support natively OTLP by this package https://github.com/openzipkin-contrib/zipkin-otel
Unfrotunately, standard distro does not have (yet?) this endpoint.

Technically, I can switch the distro, but I prefer the more OTel way with collector.

Reply:
I hope this will be sufficient 36bcd3263d8154d44dd3822e745391b3a6f14dc7